tensorflow/java/src/gen/java/org/tensorflow/processor/operator/OperatorProcessor.java

```diff
@@ -0,0 +1,115 @@
+package org.tensorflow.processor.operator;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.Filer;
+import javax.annotation.processing.Messager;
+import javax.annotation.processing.ProcessingEnvironment;
+import javax.annotation.processing.RoundEnvironment;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.tools.Diagnostic.Kind;
+
```

Comment:
Unrelated question: Why is the `RetentionPolicy` of the annotation set to `CLASS` instead of `SOURCE`?

Reply:
No good reason at this point, and we can change it to a longer retention if we find a need. Have changed it to SOURCE.